### PR TITLE
Fix to set baseLocale correctly when calculating resource path

### PR DESCRIPTION
**Pull request by gouniLee**
Fix to set baseLocale correctly when calculating resource path

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,10 @@ allows it to read and localize `appinfo.json` file. This plugin is optimized for
 
 ## Release Notes
 v1.2.11
-* updated dependencies
-* Use the logger provided by the loctool instead of using log4js directly.
-* Add node 16 version testing for circleCI (minimum version of node is v10)
+* Updated dependencies
+* Useed the logger provided by the loctool instead of using log4js directly.
```
Comment (ehoogerbeets) [inline README.md:9]: Useed -> Used

**Review by ehoogerbeets — APPROVED** [review]
